ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils_filtered_tenants.py

```diff
@@ -126,6 +127,34 @@ def _filtered_vrfs(self, tenant: dict) -> list[dict]:
                     and l3_interface.get("interfaces") is not None
                 )
             ]
+
+            vrf["_l3_multicast_enabled"] = default(get(vrf, "l3_multicast.enabled"), get(tenant, "l3_multicast.enabled"))
+
+            if vrf["_l3_multicast_enabled"] is True and not get(self._hostvars, "evpn_multicast"):
+                raise AristaAvdError(
+                    "'l3_multicast: True' is only supported on a VRF/Tenant in combination with 'underlay_multicast: True' and 'evpn_multicast: True'"
+                )
+
+            rps = []
+            for rp_address in default(get(vrf, "l3_multicast.rp_addresses"), get(tenant, "l3_multicast.rp_addresses"), []):
+                if self._hostname in get(rp_address, "nodes", default=[]) or "nodes" not in rp_address:
+                    for rp_ip in get(
+                        rp_address,
+                        "rps",
+                        required=True,
+                        org_key=f"l3_multicast.rp_addresses.rps under VRF '{vrf['name']}' in tenant '{tenant['name']}'",
+                    ):
+                        if rp_groups := get(rp_address, "groups", default=[]):
```

Comment:
The default value here is not used, since it will be evaluated to "False" hence the if will fail.

Reply:
Fixed.